### ISSUE TITLE
Label is not announced by screen readers

### ISSUE THREAD

**Issue by juuso-vaadin** [vaadin/vaadin-custom-field#90]
Focusing the inputs of custom field doesn't read out the label of the field. Component does have **aria-labelledby** attribute, but it's in light DOM while the label is within shadow DOM.

Ideally the custom field label should be read out every time an input within it is focused.

**Comment by web-padawan** [comment]
Closing in favor of https://github.com/vaadin/web-components/issues/94 which describes the same problem.
